### PR TITLE
{data}[foss/2016b,intel/2016b] ROOT v6.10.02

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/9ad266c5dbc9af5a4c944e4d9b977793 for a full test report.

File: easybuild/easyconfigs/f/FFTW/FFTW-3.3.4-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+homepage = 'http://www.fftw.org'
+description = """FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
+ in one or more dimensions, of arbitrary input size, and of both real and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = ['fftw-%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/f/FFTW/FFTW-3.3.4-intel-2016b.eb]: Please use `SOURCELOWER_TAR_GZ`

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 29 (3 easyconfigs in this PR)
node2550.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/88541056baa50bbc65053833244f032f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5a4529a231e5233dbfa09baa7886a056 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 28 out of 28 (3 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8e902e90454e5c01680eadf1b292d92c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**